src/main/java/org/sagebionetworks/web/client/widget/entity/EntityBadgeViewImpl.java

```diff
@@ -334,9 +334,7 @@ public void showUnlinkIcon() {
 		icon.setPull(Pull.RIGHT);
 		icon.setEmphasis(Emphasis.DANGER);
 		icon.addStyleName("imageButton");
-		icon.addClickHandler(event -> {
-			presenter.onUnlink();
-		});
+		icon.addClickHandler(event -> DisplayUtils.confirmDelete("Are you sure you want to delete this Link?", () -> presenter.onUnlink()));
```

Comment:
SWC-5740 - show a confirmation before unlinking an entity

I didn't realize we had `DisplayUtils.confirmDelete` until working on this. It makes this so easy, I just did it here.